.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: generic
 
+sudo: false
```

Comment:
*However*, when I look at the Travis report https://travis-ci.org/MDAnalysis/mdanalysis/jobs/244601325 it says explicitly _This job ran on our container-based trusty beta. Please read our [blog post about the public beta](http://blog.travis-ci.com/2016-11-08-trusty-container-public-beta/?utm_source=trusty-container-based-beta-notice&utm_medium=banner&utm_campaign=trusty-container-based-beta)._ so the blog instructions are not very clear.